go/oasis-node/cmd/common/common.go

```diff
@@ -201,28 +203,22 @@ func initRlimit() error {
 
 	var rlim syscall.Rlimit
 	if err := syscall.Getrlimit(syscall.RLIMIT_NOFILE, &rlim); err != nil {
-		// Log, but don't return the error, this is only used for testing
-		// and to warn the user if it's set too low.
-		rootLog.Warn("failed to query RLIMIT_NOFILE",
-			"err", err,
-		)
-		return nil
+		return fmt.Errorf("failed to query RLIMIT_NOFILE: %w", err)
 	}
 
-	if rlim.Cur <= badDefaultRlimit {
-		rootLog.Warn("the node user has a very low RLIMIT_NOFILE, consider increasing",
-			"cur", rlim.Cur,
-			"max", rlim.Max,
-		)
+	desiredLimit := viper.GetUint64(CfgDebugRlimit)
+	if flags.DebugDontBlameOasis() && desiredLimit != 0 {
```

Comment:
Should this be `desiredLimit > 0 && desiredLimit != rlim.Cur`?

Reply:
yeah i think that makes more sense, will update :+1: